controllers/utils/utils_test.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
If this test module is present, I think it should implement tests.  Otherwise I'd leave it out.

Reply:
Whoops... Didn't mean to add that in just yet.